{
  "schema_version": "1.6.3",
  "id": "UBUNTU-CVE-2025-3121",
  "details": "A vulnerability classified as problematic has been found in PyTorch 2.6.0. Affected is the function torch.jit.jit_module_from_flatbuffer. The manipulation leads to memory corruption. Local access is required to approach this attack. The exploit has been disclosed to the public and may be used.",
  "aliases": [],
  "related": [
    "CVE-2025-3121"
  ],
  "severity": [
    {
      "type": "CVSS_V3",
      "score": "CVSS:3.1/AV:L/AC:L/PR:L/UI:N/S:U/C:N/I:N/A:L"
    }
  ],
  "published": "2025-04-03T00:00:00Z",
  "modified": "2025-04-03T16:51:22Z",
  "affected": [
    {
      "package": {
        "ecosystem": "Ubuntu:22.04:LTS",
        "name": "pytorch",
        "purl": "pkg:deb/ubuntu/pytorch@1.8.1-4?arch=source&distro=jammy"
      },
      "ranges": [
        {
          "type": "ECOSYSTEM",
          "events": [
            {
              "introduced": "0"
            }
          ]
        }
      ],
      "versions": [
        "1.7.1-7",
        "1.8.1-4"
      ],
      "ecosystem_specific": {
        "ubuntu_priority": "medium"
      }
    },
    {
      "package": {
        "ecosystem": "Ubuntu:24.10",
        "name": "pytorch",
        "purl": "pkg:deb/ubuntu/pytorch@2.4.1-1ubuntu1?arch=source&distro=oracular"
      },
      "ranges": [
        {
          "type": "ECOSYSTEM",
          "events": [
            {
              "introduced": "0"
            }
          ]
        }
      ],
      "versions": [
        "2.1.2+dfsg-4ubuntu1",
        "2.1.2+dfsg-4ubuntu2",
        "2.4.1-1ubuntu1"
      ],
      "ecosystem_specific": {
        "ubuntu_priority": "medium"
      }
    }
  ],
  "references": [
    {
      "type": "REPORT",
      "url": "https://ubuntu.com/security/CVE-2025-3121"
    },
    {
      "type": "REPORT",
      "url": "https://www.cve.org/CVERecord?id=CVE-2025-3121"
    },
    {
      "type": "REPORT",
      "url": "https://github.com/pytorch/pytorch/issues/149800"
    },
    {
      "type": "REPORT",
      "url": "https://github.com/pytorch/pytorch/issues/149800#issue-2940240700"
    },
    {
      "type": "REPORT",
      "url": "https://vuldb.com/?ctiid.303012"
    },
    {
      "type": "REPORT",
      "url": "https://vuldb.com/?id.303012"
    },
    {
      "type": "REPORT",
      "url": "https://vuldb.com/?submit.525049"
    }
  ]
}